src/main/java/org/tron/core/net/node/TrxHandler.java

```diff
@@ -26,7 +26,7 @@
 
   private NodeImpl nodeImpl;
 
-  private static int MAX_TRX_SIZE = 100_000;
+  private static int MAX_TRX_SIZE = 50_000;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 50_000 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1672&issues=AWbTwVARO08i52vd_Pwl&open=AWbTwVARO08i52vd_Pwl)

[ISSUE_KEY]: AWbTwVARO08i52vd_Pwl